lang/semgrep-grammars/src/semgrep-sfapex/apex/grammar.js

```diff
@@ -19,8 +27,12 @@ module.exports = grammar(base_grammar, {
   conflicts: ($, previous) => previous.concat([
     [$.primary_expression, $.formal_parameter],
     [$.primary_expression, $.statement],
+    [$.argument_list, $.formal_parameter]
   ]),
 
+  // Originally: $.identifier
+  word: ($) => $._apex_identifier,
```

Comment:
If we don't do this, we get an error message that tells us that `identifier` must be a terminal symbol to be used as the word token. The so-called word token is set by this declaration. I had to change it because I changed the definition of `identifier`.

Reply:
I added a comment.